src/keygen/keygen_commit.rs

```diff
@@ -93,7 +89,7 @@ impl KeygenDecommit {
         sid: &Identifier,
         sender: &ParticipantIdentifier,
         com: &KeygenCommit,
-    ) -> Result<bool> {
+    ) -> Result<()> {
```

Comment:
I changed this and other remaining functions verify functions to return errors instead of bools to be consistent with the rest of the codebase

Reply:
Great! I like this better!